src/screens/RemoveCardScreen.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import Prose from '../components/Prose'
+import Main, { MainChild } from '../components/Main'
+import MainNav from '../components/MainNav'
+import Screen from '../components/Screen'
+import Button from '../components/Button'
+
+interface Props {
+  lockScreen: () => void
+}
+
+const RemoveCardScreen = ({ lockScreen }: Props) => {
+  return (
+    <Screen>
+      <Main>
+        <MainChild center>
+          <Prose textCenter>
+            <h1>Remove Card</h1>
```

Comment:
how about `Remove Card and Hand to Voter`

Reply:
👍